README.md

```diff
@@ -53,9 +53,38 @@ Name | Description
 In this simple example, we're going show you how to setup an endpoint where we can get a list of `dogs` as well as
  create and update `dogs`.
 
+ ### Create the Model
+
+ In order for the application to store and retrieve data from the database, we need to setup a model definition for `dogs`.
+ We place our model definition in `lib/models/dogs.js`. This model definition is simple - we'll
+ just save the `name` and `type` of `dog` for now.
+
+ We prefer model identities to be plural (e.g. `dogs` not `dog`) and dash-case (e.g. `dog-houses` not `dogHouses` or `dog_houses`).  When possible, the filename containing a model definition should be named with the model identity, so `lib/models/dog-houses.js` will contain the `dog-houses` model definition. It should be noted that when dealing with Waterline, this means that you'll need to exercise special care when accessing models via `request.collections()` - fetching `dog-houses` requires you to type `request.collections()['dog-houses']`.
+
+ ```js
+ 'use strict';
+
+ module.exports = {
+     identity: 'dogs',
```

Comment:
My preference is that it come from HC and we leave it out of the model def, but I didn't know if it was appropriate for this doc.  Maybe a comment indicating that it's unneccessary/unneeded because HC will infer the model name from the file name.  

Not really sure...